lesson_02/job_1/dal/local_disk.py

```diff
@@ -0,0 +1,61 @@
+import os
+import shutil
+import json
+
+
+def save_to_disk(json_content: list[dict[str, str | int]], path: str) -> None:
+    """
+    Saves JSON content to a file, cleaning the directory before saving.
+
+    Args:
+        json_content (list[dict[str, str | int]]): The JSON content to save.
+        path (str): The file path where the JSON content should be saved. Includes both the directories and the file name.
+    """
+    # Extract directory from path
+    directory = os.path.dirname(path)
+
+    # Check if the directory exists and clean it OR create it if it doesn't
+    _manage_directories(directory)
+
+    # Save the JSON content to the specified path
+    with open(path, 'w', encoding='utf-8') as f:
+        json.dump(json_content, f, ensure_ascii=False, indent=4)
+
+
+def _manage_directories(directory: str) -> None:
```

Comment:
remove the underscore, this is not private method, no need. Private its more about classes